go/service/chat_boxer.go

```diff
@@ -142,9 +142,60 @@ func (b *chatBoxer) boxMessageWithKey(msg keybase1.MessagePlaintext, key *keybas
 		KeyGeneration: key.KeyGeneration,
 	}
 
+	// sign the header, encrypted body
+	if err := b.signMessageBoxed(&boxed, signingKeyPair); err != nil {
```

Comment:
@patrickxb I moved signing related code into `chat_boxer.go` as well, her as part of boxing the message. Let me know if you see an issue here. Thanks!